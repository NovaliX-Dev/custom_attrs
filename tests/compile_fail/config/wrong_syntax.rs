#![allow(unused)]
use custom_attrs::CustomAttrs;

#[derive(CustomAttrs)]
#[attr(
    #[function]
    
    pub a: usize
)]
enum Enum {
    #[attr(a = 5)]
    Variant1,

    #[attr(a = 3)]
    Variant2,
}

fn main() {
    // let a = Enum::Variant1.get_a();
}