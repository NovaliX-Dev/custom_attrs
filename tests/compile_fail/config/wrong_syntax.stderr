error: Expected `function = ...`
 --> tests/compile_fail/config/wrong_syntax.rs:6:7
  |
6 |     #[function]
  |       ^^^^^^^^
